apps/bmd/src/APICardCatch.test.tsx

```diff
@@ -1,62 +0,0 @@
-import React from 'react'
```

Comment:
why is this being removed?

Reply:
In discussion, there were some states which could theoretically happen, but which never happened, so we decided to remove this code and related tests.